src/Drupal/Commands/core/EntityCommands.php

```diff
@@ -60,23 +64,42 @@ public function delete($entity_type, $ids = null, $options = ['bundle' => self::
                 $bundleKey = $this->entityTypeManager->getDefinition($entity_type)->getKey('bundle');
                 $query = $query->condition($bundleKey, $bundle);
             }
-            $result = $query->execute();
-            $entities = $storage->loadMultiple($result);
-        } else {
-            $entities = $storage->loadMultiple();
         }
+        $result = $query->execute();
 
         // Don't delete uid=1, uid=0.
         if ($entity_type == 'user') {
-            unset($entities[1]);
-            unset($entities[0]);
+            unset($result[0], $result[1]);
         }
 
-        if (empty($entities)) {
+        if (empty($result)) {
             $this->logger()->success(dt('No matching entities found.'));
         } else {
-            $storage->delete($entities);
-            $this->logger()->success(dt('Deleted !type entity Ids: !ids', ['!type' => $entity_type, '!ids' => implode(', ', array_keys($entities))]));
+            $chunks = $options['chunks'] ?? 50;
```

Comment:
Lets remove this line, and just default to 50 in the method params. That way the default is documented in help.

Reply:
Did know that, nice. I did the adjustment.